kubevela/kubevela
# Fix: update latest version Fix: 1.2 upgrade 1.3 workflowstep XXX not found

Signed-off-by: cezhang <c1zhang.dev@gmail.com>

handle publishversion case

Signed-off-by: cezhang <c1zhang.dev@gmail.com>


### Description of your changes



Fixes "1.2 upgrade 1.3 workflowstep XXX not found" issue

I have:


### How has this code been tested
- vela install --version 1.2.0 in k3d
- create application through CLI(explicitly define workflow) and velaux
- vela install --version 1.3.3 in k3d
- watch application running from rendering to running
- check the apprev if workstepdefinitions are patched




### Special notes for your reviewer
